src/mockServiceWorker.js

```diff
@@ -21,8 +21,13 @@ self.addEventListener('activate', async function (event) {
 })
 
 self.addEventListener('message', async function (event) {
+  if (!event.source.id) return
```

Comment:
Is it a possible scenario to receive an event without the client ID that sent it?